hack/boilerplate.go.txt

```diff
@@ -0,0 +1,15 @@
+/*
+Copyright 2021.
```

Comment:
Can you update the boilerplate with copyright owner (dunno, "Copyright 2021 The Local Storage Operator authors"?) and regenerate everything?

cc @gnufied @rohantmp @sp98 what do you think would be appropriate here?

Reply:
`Copyright 2021 The Local Storage Operator Authors` looks good to me.

Reply:
@jsafrane  @sp98  do you know how could I regenerate everything after editing `boilerplate.go.txt`?